### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: './datasets/SIDD/val/input_crops.lmdb/meta_info.txt'

### ISSUE THREAD

**Issue by leefly072** [megvii-model/HINet#31]
This is nice work and thanks for your share. however, when I train the model there have some issues, in the val dataset. I found you utilize the VAlidationNoisyBlocksSrgb.mat to validation the model. but there have some mistakes when I run the program. can you help me to solve this problem? thank you very much.

**Comment by Rookielike** [comment]
What mistakes did you meet? You can show the error message so that I can help you better

**Comment by leefly072** [comment]
When I try to train the model and  set the dataset as followed:
![image](https://user-images.githubusercontent.com/18061849/134279936-d634ef4e-b136-4020-bd14-bc0f29088cac.png)
 but there has mistake for "FileNotFoundError: [Errno 2] No such file or directory: './datasets/SIDD/val/input_crops.lmdb/meta_info.txt'"~
 Can you tell me how did you make the val dataset? Thanks very much

**Comment by Rookielike** [comment]
Did you run 'python3 scripts/data_preparation/sidd.py' to prepare the data

**Comment by leefly072** [comment]
yep, i read the code found it seems to generate the train datasets without the val dataset. Did I miss something?
![image](https://user-images.githubusercontent.com/18061849/134280673-3ae5c9e9-4a74-47c2-9e96-a72b493a836c.png)
![image](https://user-images.githubusercontent.com/18061849/134280739-e72b3641-59b4-4665-9b5a-ed86634795f5.png)

**Comment by Rookielike** [comment]
create_lmdb_for_SIDD() will generate val dataset

**Comment by leefly072** [comment]
Thanks very much, I will try to run the model "scripts/data_preparation/sidd.py'" again. And thanks for your reply!

**Comment by zhangweilong6** [comment]
have you sloved this problem?

**Comment by Pompeyhui** [comment]
原因是啥我不懂，
我的解决方案是在
[HINet/basicsr/utils/create_lmdb.py]
添加
import os
import scipy.io as scio
from tqdm import tqdm
import cv2
